benchmarks/api_benchmark.py

```diff
@@ -785,9 +785,9 @@ def body(carry):
 
   size = 100_000
   args = jnp.array([0]), jnp.zeros((1, size))
-  f(*args)  # compile
+  jax.block_until_ready(f(*args))  # compile
```

Comment:
note that this will now regress. I don't know if you wrote it, but was this the intention of the author?

Reply:
Yup, I was the author of this benchmark too. I forgot to add this in at the time, or seems!